query-engine/driver-adapters/src/proxy.rs

```diff
@@ -273,9 +275,12 @@ fn js_value_to_quaint(
             )),
         },
         ColumnType::Int64 => match json_value {
-            serde_json::Value::Number(n) => n.as_i64().map(QuaintValue::int64).ok_or(conversion_error!(
-                "number must be an i64 in column '{column_name}', got {n}"
-            )),
+            serde_json::Value::Number(n) => n
+                .as_i64()
+                .ok_or(conversion_error!(
+                    "number must be an integer in column '{column_name}', got '{n}'"
+                ))
```

Comment:
Note: I've considered extracting this into a separate function, as this logic is essentially shared with the previous `ColumnType::Int32`, `serde_json::Value::Number(n)` case, but opted against it due to Rust functions not being applicable to specific enum variants (like `serde_json::Value::Number`). Alternatively, we could store the value

```rust
conversion_error!(
    "number must be an integer in column '{column_name}', got '{n}'"
)
```

above and use twice, once for `ColumnType::Int32`, `serde_json::Value::Number(n)` and once for `ColumnType::Int64`, `serde_json::Value::Number(n)`?